src/test/java/nl/ordina/jobcrawler/scrapers/YachtVacancyScraperTest.java

```diff
@@ -18,22 +15,14 @@
 import org.springframework.web.client.RestTemplate;
 
 import java.io.File;
-import java.io.IOException;
 import java.net.URISyntaxException;
 import java.net.URL;
 import java.util.List;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNull;
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.anyInt;
-import static org.mockito.ArgumentMatchers.anyString;
-import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.spy;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
avoid wildcard imports, in intelij go to settings > Editor > Code Style > Java > increase count to use wild card imports

Reply:
Done e1130fd

Might be an idea for all of us to check these settings as I already changed them before. @narayan190290 @adriaanbakker-java